Bump acceptable versions of babl and gegl.

Patch is taken from debian:
https://packages.debian.org/stretch/gimp

Upstream-Status: Inappropriate [OE specific]
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
diff --git a/app/sanity.c b/app/sanity.c
index febddfb..85fea55 100644
--- a/app/sanity.c
+++ b/app/sanity.c
@@ -351,7 +351,7 @@ sanity_check_babl (void)
 
 #define BABL_REQUIRED_MAJOR 0
 #define BABL_REQUIRED_MINOR 1
-#define BABL_REQUIRED_MICRO 10
+#define BABL_REQUIRED_MICRO 12
 
   babl_get_version (&babl_major_version,
                     &babl_minor_version,
@@ -388,7 +388,7 @@ sanity_check_gegl (void)
   gint gegl_micro_version;
 
 #define GEGL_REQUIRED_MAJOR 0
-#define GEGL_REQUIRED_MINOR 2
+#define GEGL_REQUIRED_MINOR 3
 #define GEGL_REQUIRED_MICRO 0
 
   gegl_get_version (&gegl_major_version,
diff --git a/configure.ac b/configure.ac
index 26812b6..4252fe9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -40,8 +40,8 @@ m4_define([gimp_stable],
 m4_define([gimp_full_name], [GNU Image Manipulation Program])
 
 # required versions of other packages
-m4_define([babl_required_version], [0.1.10])
-m4_define([gegl_required_version], [0.2.0])
+m4_define([babl_required_version], [0.1.12])
+m4_define([gegl_required_version], [0.3.0])
 m4_define([glib_required_version], [2.30.2])
 m4_define([atk_required_version], [2.2.0])
 m4_define([gtk_required_version], [2.24.10])
@@ -519,7 +519,7 @@ AC_SUBST(ISO_CODES_LOCALEDIR)
 ###############################
 
 PKG_CHECK_MODULES(BABL, babl >= babl_required_version)
-PKG_CHECK_MODULES(GEGL, gegl-0.2 >= gegl_required_version)
+PKG_CHECK_MODULES(GEGL, gegl-0.3 >= gegl_required_version)
 PKG_CHECK_MODULES(ATK, atk >= atk_required_version)
 
 AM_PATH_GLIB_2_0(glib_required_version, :,
